pkg/api/common/versions.go

```diff
@@ -111,9 +111,10 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.13.2":         false,
 	"1.13.3":         false,
 	"1.13.4":         false,
-	"1.13.5":         true,
+	"1.13.5":         false,
```

Comment:
should we keep this until Azure Stack is added?

Reply:
Yes, good call. Maybe that should be our policy in general from now on: don't remove versions until Azure Stack supports two newer ones.